### DISCUSSION TITLE
Question about the validator option

### DISCUSSION THREAD

**Discussion by cbs-mrich** [moov-io/metro2#177] [Q&A]
I have been attempting to validate a metro2 file created by my software vendor.  When I use the following:
metro2.exe validator --input .\TU20230601.txt
I received the following:
Error: parsing time "        " as "01022006": cannot parse "        " as "01"

Any ideas or advice would be greatly appreciated.

**Comment by dn768** [comment]
Hey, I know it's been a while, but I just came across this. It took me a while to understand the error messages as well!

Parsing " " as "01022006" means that a date format like 01022006 is expected (Jan 2, 2006), but that there was only whitespace where the date was expected.

I've found that this metro2 project is much stricter than the credit bureaus are with compliance to the latest METRO2 format, since they tend to accept older versions.

You can check your copy of the CRRG to find the columns in which the dates are expected in the HEADER record for either packed or character format.
